### PR TITLE
[primTorch] support one tensor and two scalars in _prims.where

### PR DESCRIPTION
**Pull request by Aidyn-A**
Fixes an issue of supporting two scalar arguments for `where` and other functions with similar set of arguments:

```
refs.where(a, 1, 0)
```

I had to skip `test_python_ref_executor` because the test causes a `Segmentation fault` when running with two scalars.
The issue https://github.com/csarofeen/pytorch/issues/1770 has been fixed https://github.com/csarofeen/pytorch/pull/1774, so we can lift the skip when its merged to the upstream.

### CODE REVIEW

File: torch/testing/_internal/common_methods_invocations.py [hunk 1]
```diff
@@ -21027,6 +21034,12 @@ def __init__(
         "_refs.where",
         torch_opinfo_name="where",
         op=lambda self, condition, other: refs.where(condition, self, other),
+        skips=(
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:21037]: supports_nvfuser=False should be a more precise skip than skipping the entire executor test

**Comment by Aidyn-A** [comment]
Hi @mruberry ,

I have implemented the changes that we are agreed to do. 
Could you kindly review the PR?

**Review by ngimel — APPROVED** [review]

**Pull request closed without merging**